binaries/bzutil/main.go

```diff
@@ -130,6 +156,26 @@ func main() {
 		}
 		parseAndSetLevel(*actionLogLevel)
 		uploadBzAction(*actionAddr, *actionCommandDigest, *actionRootDigest, *actionNoCache, *actionJson)
+	} else if batchUpload.Parsed() {
+		parseAndSetLevel(*batchUploadLogLevel)
+		if strings.ToLower(strings.Trim(*batchUploadTest, " ")) == "true" && (*batchUploadDir != "") {
+			fmt.Printf("Supply either -dir or -test=true not both.\n")
+			return
```

Comment:
log.Fatal instead of returning

Reply:
done

Reply:
Another late follow up to this one - can we prevent default dir param here? If test is false, and dir is not specified - then it might just start uploading stuff in the current directory, which seems unwanted

Reply:
done